src/fidesops/ops/api/v1/endpoints/privacy_request_endpoints.py

```diff
@@ -818,6 +821,12 @@ async def resume_privacy_request_with_manual_input(
     dataset_graphs = [dataset_config.get_graph() for dataset_config in datasets]
     dataset_graph = DatasetGraph(*dataset_graphs)
 
+    if not paused_collection:
+        raise HTTPException(
+            status_code=HTTP_422_UNPROCESSABLE_ENTITY,
+            detail="Cannot save manual data on paused collection. No paused collection saved'.",
+        )
+
```

Comment:
I'm just adding a new check here now that I've generally removed the requirement that a collection is saved on a `CheckpointActionRequired` but this is one place where it's needed.